src/admin/controllers/settings/views/backups/model.php

```diff
@@ -75,26 +82,20 @@ function add_folder($zip, $folder) {
 	}
 
 	// add folders and files
-	add_folder($zip, 'admin');
-	add_folder($zip, 'controllers');
-	add_folder($zip, 'core');
+	add_folder($zip, 'src');
 	add_folder($zip, 'images');
-	add_folder($zip, 'plugins');	
-	add_folder($zip, 'templates');
-	add_folder($zip, 'user_classes');
-	add_folder($zip, 'widgets');
-	$zip->addFile(CMSPATH . "/index.php", "index.php");
-	$zip->addFile(CMSPATH . "/.htaccess", ".htaccess");
-	$zip->addFile(CMSPATH . "/cmslog.txt", "cmslog.txt");
-	$zip->addFile(CMSPATH . "/config.php", "config.php");
-	$zip->addFile(CMSPATH . "/temp/db.sql", "db.sql");
-	if (file_exists(CMSPATH . "/my_404.php")) {
-		$zip->addFile(CMSPATH . "/my_404.php", "my_404.php");
+	$zip->addFile($_ENV["backup_directory"] . "/index.php", "index.php");
+	$zip->addFile($_ENV["backup_directory"] . "/.htaccess", ".htaccess");
+	$zip->addFile($_ENV["backup_directory"] . "/cmslog.txt", "cmslog.txt");
+	$zip->addFile($_ENV["backup_directory"] . "/config.php", "config.php");
+	$zip->addFile($_ENV["backup_directory"] . "/temp/db.sql", "db.sql");
+	if(isset($_ENV["custom_404_file_path"])) {
+		$zip->addFile($_ENV["custom_404_file_path"], basename($_ENV["custom_404_file_path"]));
 	}
 	$zip->close();
 
 	// delete temp db file
 	unlink($db_backup_file);
 
-	CMS::Instance()->queue_message('Backup created','success', Config::uripath()."/admin/settings/backups");
+	CMS::Instance()->queue_message('Backup created','success', $_ENV["uripath"]."/admin/settings/backups");
```

Comment:
Worth adding backup_directory / path to the message text for clarification?

Reply:
future pr